crates/sui/tests/transaction_orchestrator_tests.rs

```diff
@@ -78,7 +80,7 @@ async fn test_blocking_execution() -> Result<(), anyhow::Error> {
     Ok(())
 }
 
-#[tokio::test]
+#[sim_test]
```

Comment:
I really dislike that more and more tests are moving to only be runnable in the simulator, can we continue having these tests run outside as well?

Reply:
I gave a pretty good reason in the PR description for moving it (5x speedup), but sure, we can use the `sui_test` macro so it runs in both.

Reply:
Do you want them to run in CI always, or do you just want the ability to run them via cargo nextest locally? I think if we move too many of them back to tokio::test in CI we'll go back to the days of frequently re-running the tests to get CI to pass.

Reply:
I don't particular care how they're run in CI, whats difficult is i'd like the default cargo test to be able to pick these up and run them locally.

Reply:
ok, I'll send a separate PR that should allow you to do that. Can you review this PR as-is please?